skfda/representation/_functional_data.py

```diff
@@ -85,6 +86,18 @@ def ndim_codomain(self):
         """
         return self.ndim_image
 
+    @property
+    @abstractmethod
```

Comment:
You can use `abstractproperty`.

Reply:
All @abstracproperties of `_functional_data.py` changed in fa02b84.

Reply:
Sorry because I was mistaken in recommending that. It seems that `abstractproperty` is in fact deprecated in favour of your initial solution 😅.

Reply:
Undo in 3707b71.